css/css-animations/timing-function-stepend-delay-ref.html

```diff
@@ -0,0 +1,12 @@
+<!doctyle html>
```

Comment:
`doctype` rather than `doctyle` (and in the reference too)... unless the test depends on being quirks mode, in which case remove the line!